src/modules/auth/index.js

```diff
@@ -439,8 +439,11 @@ export default class AuthModule extends Module {
 				timeRemaining = MAX_SESSION_DURATION;
 			}
 
+			// Check if the user is online based on presence of Ping module in the application
+			const isOnline = this.App?.AppStore?.getState()?.connectivity?.status === 'offline' ? false : true;
```

Comment:
Do we want to disable any requests to seacat auth when the application is offline?